google/auth/compute_engine/credentials.py

```diff
@@ -74,6 +75,7 @@ def __init__(
         self._scopes = scopes
         self._default_scopes = default_scopes
         self._universe_domain_cached = False
+        self._universe_domain_request = google_auth_requests.Request()
```

Comment:
Why does this request object get saved?

Reply:
All req objects are passed in except the one used in the universe_domain property. I am thinking if we need to pass in a custom req object some day, we can override this _universe_domain_request property.

Reply:
Okay gotcha